### DISCUSSION TITLE
How can this code be implemented in PK-sim?

### DISCUSSION THREAD

**Discussion by noxavier** [Open-Systems-Pharmacology/Forum#1524] [Questions & Problems]
I recently read an article on modeling the effects of [food.](https://pubs.acs.org/doi/10.1021/acs.molpharmaceut.2c00782?ref=pdf) But I don't know how to implement it in PK-sim. This is the information about the methods and supporting information in the article, and I would like to know how to implement this modification in PK-sim?
![1](https://user-images.githubusercontent.com/109507330/236981336-7d1f6cf4-7b6b-4399-840c-1f8eb8add05a.jpg)
![2](https://user-images.githubusercontent.com/109507330/236981358-c78d3685-d89e-4820-a745-081f5836948e.jpg)

**Comment by StephanSchaller** [comment]
When creating a simulation, simply head to the "Events" tab and input your preferred meal and the time you want it to take place. This will result in a change in gastric emptying following the onset of the meal:
![Uploading image.png…]()

**Comment by Yuri05** [comment]
If you need to provide units/dimension not yet available in PK-Sim, you have to modify the file 
**C:\ProgramData\Open Systems Pharmacology\PK-Sim\X.Y\OSPSuite.Dimensions.xml** and restart PK-Sim.

**Reply by Yuri05** [reply]
BTW, I assume it should be cal/min and not cal/l

**Reply by msevestre** [reply]
>https://pubs.acs.org/doi/10.1021/acs.molpharmaceut.2c00782?ref=pdf  it's in the article lol

**Reply by Yuri05** [reply]
still might be wrong

**Reply by noxavier** [reply]
> still might be wrong

Thank you for your answer. His article did not include the unit cal/l. Although I don't quite understand the meaning of this code, according to the ratio of factors in the code, it should be cal/min. Should the following J/l also be J/min? Or do you think other units are using it correctly?

**Reply by Yuri05** [reply]
>Should the following J/l also be J/min? 

Yes.
